app/Resources/views/article-text.html.twig

```diff
@@ -10,7 +10,11 @@
 
             {% if viewSelector %}
 
-                {{ render_pattern(viewSelector) }}
+                <div id="view-selector">
```

Comment:
Had a play with this, functionally it works fine and implements the behaviour that we want. I think though that you should move `id="view-selector"` into `content.html.twig`, and then remove it from both these files. I tried this out locally myself, and I think the end result is slightly better as it adds some vertical whitespace between where the focus is set and the top of the page.
This is the change I made...
`<div id="view-selector" class="wrapper{% if not isSubsequent ?? false %} wrapper--content{% endif %}">`

And this was the result which I think is visually slightly better.
![Screenshot 2021-01-07 at 08 58 03](https://user-images.githubusercontent.com/52923935/103872557-816bbb80-50c6-11eb-95c0-7e01994bd97a.png)